UniMath/Induction/README.md

```diff
@@ -0,0 +1,17 @@
+# (Co)inductive types
+
```

Comment:
I suggest you identify yourself as the author of this *package* (not directory), to take credit for it, and to be notified if changes are proposed.  If you wish ...

Reply:
@DanGrayson Sure, would you suggest removing my name/username from the individual file headers and just adding it to the README? I'm not super concerned about if/where it is listed.

Reply:
No, leave it in the individual files.  The README file is the most visible introduction to the package, so having it there (also) seems good.

Reply:
Done!